hash/bbc_flop_68000.xml

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
 
-<!-- BBC Micro - 68000 Co-Processor Discs -->
+<!-- Acorn BBC Micro - Casper 68000 Co-Processor discs -->
 
 <!-- Loading Instructions:
 
       Hold down the SHIFT key and press and release the BREAK key.
 -->
 
-<softwarelist name="bbc_flop_68000" description="Casper 68000 2nd Processor Discs">
+<softwarelist name="bbc_flop_68000" description="Acorn Casper 68000 2nd Processor discs">
```

Comment:
This implies that Acorn made the Casper 68000 boards, they didn't, it's misleading.

Reply:
By that logic I should rename bbc_flop_z80 to bbc_flop_acorn_z80, and many others, no! The torch list is a little different as Torch made a combined Z80/68000 board (not yet emulated), so contains software for Torch machines with both co-processors.

Reply:
OK, I agree. Thanks for clarification :)